src/main/java/us/ihmc/mecano/algorithms/JointTorqueRegressorCalculator.java

```diff
@@ -3,19 +3,13 @@
 import org.ejml.data.DMatrix;
 import org.ejml.data.DMatrixRMaj;
 import org.ejml.dense.row.CommonOps_DDRM;
-import us.ihmc.mecano.multiBodySystem.interfaces.JointReadOnly;
-import us.ihmc.mecano.multiBodySystem.interfaces.MultiBodySystemBasics;
-import us.ihmc.mecano.multiBodySystem.interfaces.MultiBodySystemReadOnly;
-import us.ihmc.mecano.multiBodySystem.interfaces.RigidBodyBasics;
+import us.ihmc.mecano.multiBodySystem.interfaces.*;
```

Comment:
No need to do anything in this PR, but don't forget to update your intellij java code style to the most recent one

Reply:
👍🏻 have done.